src/target/target.c

```diff
@@ -744,16 +744,22 @@ static int default_check_reset(struct target *target)
  * Keep in sync */
 int target_examine_one(struct target *target)
 {
+	if (target->examine_attempted)
+		LOG_TARGET_INFO(target, "Retry examination.");
+
+	LOG_TARGET_INFO(target, "Examination started.");
+
 	target_call_event_callbacks(target, TARGET_EVENT_EXAMINE_START);
 
 	int retval = target->type->examine(target);
 	if (retval != ERROR_OK) {
+		LOG_TARGET_ERROR(target, "Examination failed. examine() -> %d", retval);
 		target_reset_examined(target);
 		target_call_event_callbacks(target, TARGET_EVENT_EXAMINE_FAIL);
 		return retval;
 	}
 
-	LOG_USER("[%s] Target successfully examined.", target_name(target));
+	LOG_TARGET_INFO(target, "Target successfully examined.");
```

Comment:
Why are you changing this from _USER to _INFO? (I don't actually know what the difference is.)

Reply:
Difference that output of `LOG_USER` will be showed with any `debug_level`, `LOG_INFO` will be showed with `debug_level 2`. I added message `Examination started` using `LOG_TARGET_INFO` so I thought that is good idea to use `LOG_TARGET_INFO` for this message too,

Reply:
OK